ebean-core/src/main/java/io/ebeaninternal/server/core/DefaultServer.java

```diff
@@ -246,7 +246,7 @@ private void configureServerPlugins() {
    */
   public void executePlugins(boolean online) {
     if (!config.isDocStoreOnly()) {
-      ddlGenerator.execute(online);
+      ddlGenerator.generateDdl();
```

Comment:
So DDL used to run before the plugins were executed and now that doesn't happen. I don't think this is correct.